main.py

```diff
@@ -496,20 +495,17 @@ def set_tag_range(self, request: Request) -> JSONResponse:
         tag_type = content.get("tag_type")
         try:
             ranges = get_tag_ranges(content["tag_ranges"])
-        except KytosInvalidRanges as err:
-            raise HTTPException(400, detail=err)
+        except KytosInvalidTagRanges as err:
+            raise HTTPException(400, detail=str(err))
         interface_id = request.path_params["interface_id"]
         interface = self.controller.get_interface_by_id(interface_id)
         if not interface:
             raise HTTPException(404, detail="Interface not found")
         try:
             interface.set_tag_ranges(ranges, tag_type)
             self.handle_on_interface_tags(interface)
-        except KytosSetTagRangeError as err:
-            detail = f"The new tag_ranges cannot be applied {err}"
-            raise HTTPException(400, detail=detail)
-        except KytosTagtypeNotSupported as err:
-            detail = f"Error with tag_type. {err}"
+        except KytosTagError as err:
+            detail = f"Error when setting tag ranges: {err}"
```

Comment:
Cool to have `KytosTagError` here. Since all the child exceptions have great names and the `__repr__` is also including its name and the message, maybe in the detail you can just pass `str(err)`? Check out the current error below, since `"Error when setting tag ranges:"` is being concatenated it might get repetitive since `KytosSetTagRangeError` will be in the message, see what I mean, check this example below? Trivial stuff though.  

![20231109_130214](https://github.com/kytos-ng/topology/assets/1010796/b719bcf2-3a83-4a4f-ba55-4851d79c3fc7)

Reply:
Oh got it, lines 508 and 525 changed.